src/docs/doxygen/mainpage.md

```diff
@@ -0,0 +1,23 @@
+# Source Code Documentation #
+
+#### Physics simulation modules ####
```

Comment:
These should be capitalized:

"Physics Simulation Modules"
"Simulation Utilities"
"Numerics Utilities"